src/components/utils/toolbar.ts

```diff
@@ -0,0 +1,20 @@
+const texts = {
+    showMore: 'Se alle',
+    showLess: 'Vis færre',
+};
+
+export const PAGINATED_NUMBER_OF_ITEMS = 50;
+
+export const onTogglePaginationClick = (numberOfItemsPerPage: number, setNumberOfItemsPerPage: (n: number) => void, numberOfItemsTotal: number) => {
```

Comment:
Er disse funksjonene er definert to steder nå?

Reply:
Uff, stemmer. Jeg fjerner toolbar.ts, da den ikke lenger er i bruk.